subprojects/packagefiles/lcms2/meson.build

```diff
@@ -0,0 +1,65 @@
+project('Little-CMS', 'c',
+  version: '2.12'
+)
+
+cc = meson.get_compiler('c')
+
+lcms2_srcs = [ 'src/cmsalpha.c'
+        , 'src/cmscgats.c'
+        , 'src/cmserr.c'
+        , 'src/cmsgmt.c'
+        , 'src/cmsintrp.c'
+        , 'src/cmsio1.c'
+        , 'src/cmsmd5.c'
+        , 'src/cmsnamed.c'
+        , 'src/cmspack.c'
+        , 'src/cmsplugin.c'
+        , 'src/cmssamp.c'
+        , 'src/cmstypes.c'
+        , 'src/cmswtpnt.c'
+        , 'src/cmscam02.c'
+        , 'src/cmscnvrt.c'
+        , 'src/cmsgamma.c'
+        , 'src/cmshalf.c'
+        , 'src/cmsio0.c'
+        , 'src/cmslut.c'
+        , 'src/cmsmtrx.c'
+        , 'src/cmsopt.c'
+        , 'src/cmspcs.c'
+        , 'src/cmsps2.c'
+        , 'src/cmssm.c'
+        , 'src/cmsvirt.c'
+        , 'src/cmsxform.c'
+]
+
+inc_dirs = include_directories('include')
+
+deps = [
+  cc.find_library('m', required: false),
+  dependency('threads'),
+  dependency('zlib'),
+  dependency('libtiff-4'),
+]
+
+cargs = ['-DHasTHREADS=1', '-DHasZLIB=1', '-DHasTIFF=1']
+
+if cc.has_argument('-fvisibility')
+  cargs +='-DHAVE_FUNC_ATTRIBUTE_VISIBILITY=1'
+elif cc.get_id() == 'msvc'
+  cargs += '-DCMS_DLL_BUILD=1'
+endif
+
+liblcms2_lib = library('lcms2', lcms2_srcs,
```

Comment:
Also the autotools build creates a versioned library "2.0.12" (I believe due to misunderstanding libtool, but everyone misunderstands that anyway).

Reply:
done.

Reply:
nooooooooo the version has the 0 in the middle :p

(Yes I know it makes no sense and goes completely against the actual project version.)

Also: version, not soversion, the soversion is the part encoded in the DT_NEEDED ("Library soname") while version is the filename and can set the soversion automatically from the first component of the version.